### PR TITLE
Axosoft.GitKraken version 9.2.1

### LINKED ISSUE #101499: [Update Request]: GitKraken - 9.2.1
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Axosoft.GitKraken

### Package Version

9.2.1

### Please describe the changes you would like to see

There is a new release (9.2.1) available
Download link - https://www.gitkraken.com/download

### PR DESCRIPTION
**Pull request by afermar**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----
Resolves #101499 

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/101621)

### CODE REVIEW

File: manifests/a/Axosoft/GitKraken/9.2.1/Axosoft.GitKraken.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  InstallerUrl: https://release.axocdn.com/win64/GitKrakenSetup-9.2.1.exe
+  InstallerSha256: 157789AA51B1087BDD6A74231CF21E01F4E455729F3837D74F2FEF0E5DE94067
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://release.axocdn.com/win32/GitKrakenSetup-9.2.1.exe
+  InstallerSha256: DAB184393D1C734C03F64BA0BF6B85C99B10A39C2545EDC5C440497A3CC654D1
```
Comment (BrandonWanHuanSheng) [inline manifests/a/Axosoft/GitKraken/9.2.1/Axosoft.GitKraken.installer.yaml:26]: SHA256 Mismatch: 
Correct Sha256 is B0536BFDF0808701667F5B22CA088F4583E4BD86FA2CF0AC6950638DBDC73FC5

**Review by BrandonWanHuanSheng — COMMENTED** [review]
Azure Pipeline say you have a file mismatch on x86.

**Comment by BrandonWanHuanSheng** [comment]
I don't have write access. Normally, Moderator just only have a write access to the folder. The person who is moderating in the last. Thank you.

**Review by BrandonWanHuanSheng — COMMENTED** [review]
Right now the manifest is fine but I need to wait for moderator to complete the change because I don't have write access to the master.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**